test/acceptance/georeference-street-address.js

```diff
@@ -50,75 +48,143 @@ describe('georeference-street-address analysis', function() {
-                city: 'Logroño'
-            }),
-            template: '{{city}}, Argentina',
-            x: -61.69614,
-            y: -29.50347
+            template: '{{city}}, Spain',
+            query: 'select * from georeference_street_address_fixture where cartodb_id = 6',
+            addresses: [{
+                street_name: 'Logroño',
+                city: 'Logroño',
+                point: {
+                    x: -2.517555,
+                    y: 42.302939
+                },
+            }],
         },
         {
             desc: 'template with spaces in token',
-            addressNode: addressSourceNode({
-                city: 'Logroño'
-            }),
-            template: '{{ city }}, Argentina',
-            x: -61.69614,
-            y: -29.50347
+            template: '{{ city  }}, La Rioja, Spain',
+            query: 'select * from georeference_street_address_fixture where cartodb_id = 6',
+            addresses: [{
+                street_name: 'Logroño',
+                city: 'Logroño',
+                point: {
+                    x: -2.517555,
+                    y: 42.302939
+                },
+            }],
         },
         {
             desc: 'with column and more free text',
-            addressNode: addressSourceNode({
-                city: 'Logroño'
-            }),
             template: '{{city}}, La Rioja, Spain',
-            x: -2.517555,
-            y: 42.302939
+            query: 'select * from georeference_street_address_fixture where cartodb_id = 6',
+            addresses: [{
+                street_name: 'Logroño',
+                city: 'Logroño',
+                point: {
+                    x: -2.517555,
+                    y: 42.302939
+                },
+            }]
         },
         {
             desc: 'with several columns and free text',
-            addressNode: addressSourceNode({
-                city: 'Logroño',
-                state: 'La Rioja'
-            }),
             template: '{{city}}, {{state}}, Spain',
-            x: -2.517555,
-            y: 42.302939
+            query: 'select * from georeference_street_address_fixture where cartodb_id = 4',
+            addresses: [{
+                point: {
+                    x: -2.517555,
+                    y: 42.302939
+                }
+            }]
         },
         {
             desc: 'with only free text',
-            addressNode: addressSourceNode(),
             template: 'Logroño, La Rioja, Spain',
-            x: -2.517555,
-            y: 42.302939
+            query: 'select * from georeference_street_address_fixture where cartodb_id = 1',
+            addresses: [
+                { cartodb_id: 1, point: { x: -2.517555, y: 42.302939 }, street_name: 'W 26th Street' }
+            ]
+        },
+        {
+            desc: 'multiple rows',
+            column: 'full_address',
+            query: 'select * from georeference_street_full_address_fixture',
```

Comment:
Please use the clause `ORDER BY` for this test. This test against `< Postgres@10` will fail because the default order has changed for `Postgres@10` as the assertion phase will loop over the result in the very same order that it was queried.